### ISSUE TITLE
"Pass is not initialized" on LLVM 8

### ISSUE THREAD

**Issue by easyaspi314** [Wilfred/bfc#27]
```
Pass 'Interprocedural Sparse Conditional Constant Propagation' is not initialized.
Verify if there is a pass dependency cycle.
Required Passes:
Segmentation fault: 11
```

Even when I updated llvm-sys to v80, it just crashes.

**Comment by Wilfred** [comment]
What platform are you running on? How are you running bfc?

**Comment by easyaspi314** [comment]
Oof, I put the info in the wrong tab. The side effect of having 201 tabs open. >.<

macOS Mojave 10.14.2
2.0 GHz MacBook Pro (15-inch, Early 2011)
Clang 8.0.0 from Homebrew
[This file](https://github.com/erikdubbelboer/brainfuck-jit/blob/master/mandelbrot.bf) 

```
../bfc/target/release/bfc -O2 ../mandelbrot.bf
```
If I compile against Clang 7.0.1, it works fine.

I tried to compile for Termux on ARMv7-A, but I got a bunch of linker errors. I'll try to reproduce on Linux.

**Comment by apaprocki** [comment]
@Wilfred I see this happening on a vanilla x86_64 RHEL server as well with Clang 8.0.0.

**Comment by BambooL** [comment]
Same issue here. vanilla x86_64 with llvm@8. Same error message with every single compile.

**Comment by Wilfred** [comment]
I can reproduce this on release builds on bfc, but not debug builds.

**Comment by Wilfred** [comment]
I specifically said "doesn't fix" in the commit message!

**Comment by Wilfred** [comment]
The above commit should fix it, let me know if not :)

**Comment by apaprocki** [comment]
@Wilfred thanks, that does it. Is there an expected date for the 1.8.0 release? If you're holding off a bit, I'll pull the patches out. Thanks

**Comment by Wilfred** [comment]
I've just tagged 1.8.0 and published it. I screwed up slightly and the published release is 700a2ae, but hopefully that should meet your needs.

I'm flattered and rather surprised that people actually want releases of this project! It was purely intended as a {well,over}engineered toy.

**Comment by BambooL** [comment]
Thanks Wilfred,

for addressing this issue and releasing a new version!


Xiao
